docs/10X_alignment.md

```diff
@@ -1,21 +1,15 @@
-# 10X workflow
+# 10X Alignment Workflow
 
-The workflow script that runs the tools is `workflows/kf_single_cell_10x.cwl`
+The workflow script that runs the tools is `workflows/kf_single_cell_10x_alignment.cwl`
 
 The workflow runs [cellranger count](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/6.0/using/count),
 on fastq files generated by the 10x single cell RNA workflow methodology.
 Cell ranger count performs alignment, barcode counting, and filtering.
-[SoupX](https://github.com/constantAmateur/SoupX) is used for subtraction of the RNA background.
-[Scrublet](https://github.com/swolock/scrublet) is used to score and predict doublets.
-Decontaminated outputs are aggregated using the [Seurat](https://satijalab.org/seurat/) R package from the Satija lab at the New York Genome Center.
-Original workflow design heavily contributed to by Erin Reichenbee of DBHi.
 A custom QC R markdown notebook developed by @AntoniaChroni is also run, which includes as it's main engine Seurat and [scooter](https://github.com/igordot/scooter)
 
-## Tools Ran
+## Software
 
 - Cellranger 6.0.0
```

Comment:
No too big a deal, but it;s actually `6.1.2`:
`pgc-images.sbgenomics.com/d3b-bixu/cellranger:6.1.2`